sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/QueryBuilderSamples.cs

```diff
@@ -57,6 +57,20 @@ public static void Main()
                 .Select("Room", "Temperature")
                 .From(AdtCollection.DigitalTwins)
                 .Build();
+
+            // SELECT * FROM DIGITALTWINS WHERE TEMPERATURE < 5
+            AdtQueryBuilder query6 = new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .WhereComparison("Temperature", QueryComparisonOperator.LessThan, "6")
```

Comment:
I wonder if the name of the method can just be reduced to `Where` and let the method signature dictate the possible inputs.

Reply:
Assuming we move to the style `.Where().Comparison(...)` I wouldn't .